package.json

```diff
@@ -115,9 +116,13 @@
   },
   "dependencies": {
     "commander": "2.15.1",
+    "debug": "^3.1.0",
+    "dotenv": "^6.0.0",
     "form-data": "2.3.2",
+    "getenv": "^0.7.0",
     "minimist": "1.2.0",
     "node-fetch": "2.1.2",
-    "resolve-cwd": "2.0.0"
+    "resolve-cwd": "2.0.0",
+    "supports-color": "^5.5.0"
```

Comment:
Is that used?

Reply:
Yep, its a peer dependency that `debug` likes to have